### DISCUSSION TITLE
DiceBear Editor - Beta

### DISCUSSION THREAD

**Discussion by FlorianKoerner** [dicebear/dicebear#286] [Announcements]
[![preview](https://user-images.githubusercontent.com/647303/219900272-378d48f2-5384-418f-8f95-0165bd0eae79.png)](https://editor.dicebear.com)

Today I am very happy to present you the new Avatar Editor! The editor allows you to create your own avatars and is available at [editor.dicebear.com](https://editor.dicebear.com). The editor is currently in beta and completely [open source](https://github.com/dicebear/dicebear/tree/9.x/apps/editor). Everyone is welcome to participate in the development of the editor and contribute their own ideas.

Right now the editor is limited to creating and downloading avatars, but I'm working on extending it so that it can be integrated into your own website and allow users to create their own personalized avatars.

I'm very excited to share this editor with you and I'm eager to hear your feedback. If you have any ideas or suggestions, please don't hesitate to share them with me. Together we can make this tool even better!

**Comment by SnowingFox** [comment]
Do you have plans to add [Identicon](https://dicebear.com/styles/identicon) to the editor?

**Reply by FlorianKoerner** [reply]
Primarily, the editor should make the avatar styles customizable that show a character.

But, I can easily integrate the more abstract avatar styles.

What settings should be available for the more abstract avatars? Is color and the random button enough?

**Comment by nohehf** [comment]
Hey @FlorianKoerner glad you released the editor!
I think a great way to rapidly integrate the editor into apps workflows is to allow passing some parameters in the query params (such as a fixed style), and then to make the save button redirect to some callback URL. I'll be happy to contribute to the design and development of the feature.

**Reply by FlorianKoerner** [reply]
That is exactly what I have planned. 👍 I'm working on it now and will give an update as soon as possible.

**Reply by nohehf** [reply]
Oh, nice! Lmk if you need help / when you're ready to receive PRs

**Comment by samuelkarani** [comment]
You can check  out my editor version in React & Typescript that uses 2-level navigation but supports only _Avataaars_ style: https://stackblitz.com/edit/vitejs-vite-blnx4w?file=README.md
